contributors/devel/development.md

```diff
@@ -259,6 +259,9 @@ cd $working_dir/kubernetes
 make
 ```
 
+(Note that this requires an unset `CDPATH` environment variable or one starting
```

Comment:
```
*Note:* If you are using `CDPATH`, you must either start it with a leading colon, or unset the variable. The make rules and scripts to build require the current directory to come first on the CD search path.
```

Feel free to tweak it a bit more, but we want to:
- Start off with the conditional as many people will not even have `CDPATH` set in the first place, so we don't need to state it's a requirement before we establish that they even have `CDPATH` set.
- Explain why this is needed.